_includes/v1.1/extensions-v1beta1-definitions.html

```diff
@@ -1745,14 +1745,14 @@ <h3 id="_v1_container">v1.Container</h3>
+<td class="tableblock halign-left valign-top"><p class="tableblock">Entrypoint array. Not executed within a shell. The docker image’s entrypoint is used if this is not provided. Variable references $(VAR_NAME) are expanded using the container’s environment. If a variable cannot be resolved, the reference in the input string will be unchanged. The $(VAR_NAME) syntax can be escaped with a double , ie: (VAR_NAME). Escaped references will never be expanded, regardless of whether the variable exists or not. Cannot be updated. More info: <a href="http://kubernetes.io/v1.1/docs/user-guide/containers.html#containers-and-commands">http://kubernetes.io/v1.1/docs/user-guide/containers.html#containers-and-commands</a></p></td>
 <td class="tableblock halign-left valign-top"><p class="tableblock">false</p></td>
 <td class="tableblock halign-left valign-top"><p class="tableblock">string array</p></td>
 <td class="tableblock halign-left valign-top"></td>
 </tr>
 <tr>
 <td class="tableblock halign-left valign-top"><p class="tableblock">args</p></td>
-<td class="tableblock halign-left valign-top"><p class="tableblock">Arguments to the entrypoint. The docker image’s cmd is used if this is not provided. Variable references $(VAR_NAME) are expanded using the container’s environment. If a variable cannot be resolved, the reference in the input string will be unchanged. The $(VAR_NAME) syntax can be escaped with a double $$, ie: $$(VAR_NAME). Escaped references will never be expanded, regardless of whether the variable exists or not. Cannot be updated. More info: <a href="http://kubernetes.io/v1.1/docs/user-guide/containers.html#containers-and-commands">http://kubernetes.io/v1.1/docs/user-guide/containers.html#containers-and-commands</a></p></td>
+<td class="tableblock halign-left valign-top"><p class="tableblock">Arguments to the entrypoint. The docker image’s cmd is used if this is not provided. Variable references $(VAR_NAME) are expanded using the container’s environment. If a variable cannot be resolved, the reference in the input string will be unchanged. The $(VAR_NAME) syntax can be escaped with a double , ie: (VAR_NAME). Escaped references will never be expanded, regardless of whether the variable exists or not. Cannot be updated. More info: <a href="http://kubernetes.io/v1.1/docs/user-guide/containers.html#containers-and-commands">http://kubernetes.io/v1.1/docs/user-guide/containers.html#containers-and-commands</a></p></td>
```

Comment:
We will need to fix this the right way (https://github.com/kubernetes/kubernetes/pull/17299#issuecomment-161187211) so that it is not reverted next time.
cc @caesarxuchao